LICENSE.txt

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2016 OSS Friday
+Copyright (c) 2016 Open Source Friday
```

Comment:
FWIW, "Open Source Friday" isn't an entity that can own a copyright. "Open Source Friday [contributors](https://github.com/ossfriday/ossfriday/graphs/contributors)" is probably more appropriate.